### PR TITLE
Inband

### PR DESCRIPTION
**Pull request by joelrebel**
#### What does this PR do

- Includes component Capability data.
- Adds the `alloy version` command.
- Updates build release process to set version information on the executable.
- Updates Alloy for inband data collection.
- Fixes and updates test fixtures to include namespace data.
- Fixes a bug in the namespace name.

### CODE REVIEW

File: internal/model/serverservice.go [hunk 1]
```diff
@@ -36,12 +36,12 @@ const (
 
 // ServerServiceAttributeNS returns the namespace server component attributes are stored in.
 func ServerComponentAttributeNS(appKind string) string {
-	return fmt.Sprintf("%s.alloy.%s.metadata", ServerServiceNSPrefix, appKind)
+	return fmt.Sprintf("%s.%s.metadata", ServerServiceNSPrefix, appKind)
 }
 
 // ServerComponentVersionedAttributeNS returns the namespace server component versioned attributes are stored in,
```
Comment (diogomatsubara) [inline internal/model/serverservice.go:42]: s/in,/in./

**Review by diogomatsubara — APPROVED** [review]
Few small comments otherwise looks good to me.

File: internal/fixtures/serverservice_components_E3C246D4INL.go [hunk 2]
```diff
@@ -55,7 +55,7 @@ var (
 			Serial: "0",
 			Attributes: []serverservice.Attributes{
 				serverservice.Attributes{
-					Namespace: "",
+					Namespace: "sh.hollow.alloy.outofband.metadata",
```
[the remark this answers is not in the document]
Comment (joelrebel) [reply internal/fixtures/serverservice_components_E3C246D4INL.go:58]: Yep - `inband` and `outofband` Attributes, VersionedAttributes (they share the Vendor, Model, Serial field for components) will be given their own namespaces.  Flasher will look at `outofband` data for `outofband` updates and `inband` when applying `inband` updates (when it gets to that). Its upto clients to decide which data they want to work with.

For some components the `inband` data has more resolution compared to the `outofband` data, and vice versa for `outofband`, and so we leave this choice to the consumers of the data

**Review by diogomatsubara — APPROVED** [review]
LGTM

**Pull request merged**